.github/workflows/create-release.yml

```diff
@@ -0,0 +1,22 @@
+name: Create GitHub release
+
+on:
```

Comment:
We'll be changing this trigger to use `repository_dispatch` so we can trigger it with HTTP requests sent from Blue 🐦